lacework-agent/values.yaml

```diff
@@ -35,14 +35,15 @@ clusterAgent:
   image:
     registry: docker.io
     repository: lacework/k8scollector
-    tag: latest
+    tag: 6.1.0
```

Comment:
@rnalexlacework @cirego - shall I leave this tag as unaltered (i.e. leave it as `latest`?)

Reply:
My guess is we should not pin it to 6.1.0.  May be @cirego or @nschmeller have a better idea.